orchestra/static/orchestra/todos/todo-list.directive.es6.js

```diff
@@ -76,6 +87,16 @@ export default function todoList (orchestraApi) {
         })
       }
 
+      todoList.addTodoListTemplate = () => {
```

Comment:
is this redundant with `addTodosFromTodoListTemplate` Do we only need one?

Reply:
Well, I think I over-engineered for a scenario when `addTodosFromTodoListTemplate` might be reused. I cannot think of any at this point. I will just make it one function for now.